core/src/main/scala/io/circe/sbt/CirceOrgScalaJSPlugin.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2022 circe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.circe.sbt
+
+/**
+ * automatically loaded for all projects that are cross-compiled to ScalaJS, no need too reference it explicitly in builds
+ */
+
+import org.scalajs.sbtplugin.ScalaJSPlugin
+import sbt.*
+import scoverage.ScoverageSbtPlugin.autoImport._
+
+object CirceOrgScalaJSPlugin extends AutoPlugin {
```

Comment:
Maybe we should also add a Scala Native counterpart of this plugin.

Reply:
Added one